ci-operator/step-registry/nutanix/provision/bastionhost/nutanix-provision-bastionhost-commands.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+function upload_rhcos_image() {
+   curl -X POST --header "Content-Type: application/json" \
+      --header "Accept: application/json" \
+      --header "Authorization: Basic ${ENCODED_CREDS}" \
+      "https://${NUTANIX_HOST}:9440/api/nutanix/v3/images" \
```

Comment:
I guess we can use `NUTANIX_PORT` instead of `9440`, right?

Reply:
Good catch, I'll replace them with NUTANIX_PORT